app/connectors/aws/S3FileManager.scala

```diff
@@ -119,4 +116,21 @@ class S3FileManager @Inject()(s3Client: AmazonS3, config: ServiceConfiguration)
       ObjectMetadata(metadata.getUserMetadata.asScala.toMap, metadata.getLastModified.toInstant)
     }
   }
+
+  override def getObjectContentAsByeArray(objectLocation: S3ObjectLocation): Future[ObjectContentAsByteArray] = {
+    implicit val ld = LoggingDetails.fromS3ObjectLocation(objectLocation)
+
+    Future {
+      val fileFromLocation = s3Client.getObject(objectLocation.bucket, objectLocation.objectKey)
+
+      val objectContent = fileFromLocation.getObjectContent
+      val objectContentAsByteArray =
+        Stream.continually(objectContent.read).takeWhile(_ != -1).map(_.toByte).toArray
```

Comment:
I'm not sure about this change. You now load entire files to memory. Also it seems that you were already closing the stream by doing `inputStream.close()` in class `TikaFileTypeDetector`